content/rs/installing-upgrading/supported-platforms.md

```diff
@@ -27,3 +27,20 @@ Make sure your system meets these requirements:
 | Amazon Linux | Version 1 |
 | Docker | Redis Enterprise Software Docker images are certified for Development and Testing only. |
 | Kubernetes, Pivotal Platform (PCF) and other orchestration and cloud environments | See the [Platform documentation]({{< relref "/platforms" >}}) |
+
+## VMWare
+
+Redis Enterprise is compatible with VMware.
+
+However, when deploying on VMWare, you must carefully configure your memory, CPU,
```

Comment:
I suggest we link to:
https://docs.redislabs.com/latest/rs/administering/designing-production/hardware-requirements/